app/controllers/posts_controller.rb

```diff
@@ -10,4 +10,30 @@ def show
     @post = Post.find(params[:id])
     @post_comments = Comment.where(post_id: @post)
   end
+
+  def new
+    @post = Post.new
+  end
+
```

Comment:
- [ ] It would be nice if you fix the functionality of your post form to make your project work as expected in the project requirements :+1: Currently, i am not able to create any post from your post form :fearful: